src/main/java/ui/BoardAutoCreator.java

```diff
@@ -74,6 +87,79 @@ private void createMilestoneBoard() {
                         "It is saved under the name \"" + boardName + "\".");
     }
 
+    private void createWorkAllocationBoard() {
+        logger.info("Creating " + WORK_ALLOCATION + " board");
+
+        List<TurboUser> userList = ui.logic.getRepo(ui.logic.getDefaultRepo()).getUsers();
+
+        if (userList.isEmpty()) {
+            DialogMessage.showInformationDialog("Auto-create Board - " + WORK_ALLOCATION,
+                    WORK_ALLOCATION + " board cannot be created as your" +
+                            " account has no push access to the default repo.");
+
+            return;
+        }
+
+        panelControl.closeAllPanels();
+
+        FilterPanel panelAdded;
```

Comment:
I will move `closeAllPanels()` to inside `createBoard()`. #1077 should watch out for this.